src/ship/Propulsion.h

```diff
@@ -116,7 +116,7 @@ class Propulsion : public RefCounted {
 	bool AIMatchVel(const vector3d &vel, const vector3d &powerLimit = vector3d(1.0));
 	bool AIChangeVelBy(const vector3d &diffvel, const vector3d &powerLimit = vector3d(1.0)); // acts in object space
 	vector3d AIChangeVelDir(const vector3d &diffvel);										 // object space, maintain direction
-	void AIMatchAngVelObjSpace(const vector3d &angvel, const vector3d &powerLimit = vector3d(1.0));
+	void AIMatchAngVelObjSpace(const vector3d &angvel, const vector3d &powerLimit = vector3d(1.0), bool allAxes = true);
```

Comment:
Just a thought, I personally would like a more descriptive name for this bool, like `ignoreZeroValues` or something like that.

Reply:
OK